activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ModelServiceImpl.java

```diff
@@ -450,17 +451,24 @@ public void validateModelContent(Model model,
     @Override
     public void validateModelContent(Model model,
                                      FileContent fileContent) {
-        ValidationContext validationContext = !modelTypeService.isJson(findModelType(model)) && fileContent.getContentType().equals(CONTENT_TYPE_JSON)
-                ? EMPTY_CONTEXT
-                : createValidationContext(model);
-
+        ValidationContext validationContext = getValidationContext(model, fileContent, null);
         validateModelContent(model.getType(),
                              fileContent.getFileContent(),
                              validationContext);
     }
 
 
 
+    private ValidationContext getValidationContext(Model model, FileContent fileContent, @Nullable Project project) {
+        if (!modelTypeService.isJson(findModelType(model)) && fileContent.getContentType().equals(CONTENT_TYPE_JSON))
```

Comment:
Please, always use brackets for the body of an `if` statement, even if it contains only a single line.